golang/.idea/golang.iml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you delete this .idea folder altogether?

Reply:
I removed the folder.